redaxo/src/core/lib/sql/sql.php

```diff
@@ -1019,12 +1060,16 @@ public function getArray($query = null, array $params = [], $fetchType = PDO::FE
 
     /**
      * Gibt die zuletzt aufgetretene Fehlernummer zurueck.
+     * @return mixed|string
```

Comment:
wieso mixed?

Reply:
weil `self::$pdo[$this->DBID]->errorCode()` mixed liefert

Reply:
laut [php.net](http://php.net/manual/de/pdo.errorcode.php) liefert die Methode nur String

Reply:
da sagt meine IDE etwas anderes, die hat auch die return annotation gesetzt.